aws/package.json

```diff
@@ -26,6 +26,7 @@
         "typescript": "^2.6.2"
     },
     "peerDependencies": {
-        "@pulumi/cloud": "${VERSION}"
+        "@pulumi/cloud": "${VERSION}",
+        "@pulumi/docker": "${VERSION}"
```

Comment:
this needs to be a real dependency.  but we need @pulumi/docker actually publishing first.  on @ellismg to get that up and running today.